Dockerfile.template

```diff
@@ -58,7 +57,8 @@ RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends \
 	curl -sSL $downloadURL | tar -xz -C ~/ruby --strip-components=1 && \
 	cd ~/ruby && \
 	autoconf && \
-	./configure --enable-yjit --enable-shared && \
+	# NOTE: YJIT is not available in Ruby 3.0
+	./configure --enable-shared && \
```

Comment:
One last change request, can you readd the YJIT installation and deps, on Ruby versions lower than 3.2 the build ignores the attempt to build YJIT support:

`
docker run -it cimg/ruby:3.1 ruby --enable-yjit -v
ruby 3.1.4p223 (2023-03-30 revision 957bb7cb81) [aarch64-linux]
Versus 3.2 (note +YJIT)

docker run -it cimg/ruby:3.2 ruby --enable-yjit -v
ruby 3.2.2 (2023-03-30 revision e51014f9c0) +YJIT [aarch64-linux]`

I believe it is best to leave it here in the `dockerfile.template` and allow it to be ignored so that these changes don't accidentally bleed into newer versions